### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by yasoob** [godtoy/akamai-web-sensor-mock#1]
Any plans for adding a readme with some details on how to use this? Does this bypass Akamai bot detection in Nike login?

**Comment by sriramkumar1996** [comment]
After git clone, try to run the code as below:

1. In rename the `web.akamai.ts` file to `web_akamai.ts` (or something else)
2. In `webgl.ts` file, add the below code

```
import { WebAkamai } from './web_akamai';

let akamai = new WebAkamai();
// console.log(akamai.bmak);
let output = akamai.create();
console.log(output);
``` 

3. Compile the `webgl.ts` file as `tsc webgl.ts`
4. Run the generated js file as `node webgl.js`

**Comment by DontAskM8** [comment]
> After git clone, try to run the code as below:
> 
> 1. In rename the `web.akamai.ts` file to `web_akamai.ts` (or something else)
> 2. In `webgl.ts` file, add the below code
> 
> ```
> import { WebAkamai } from './web_akamai';
> 
> let akamai = new WebAkamai();
> // console.log(akamai.bmak);
> let output = akamai.create();
> console.log(output);
> ```
> 
> 1. Compile the `webgl.ts` file as `tsc webgl.ts`
> 2. Run the generated js file as `node webgl.js`

damn idek how to use this but now its working like a charm

**Comment by skdcodes** [comment]
This script is outdated. Akamai has updated the script. Even though this script might generate "some" `sensor_data`, it will eventually get rejected if you send a real-time request.

**Comment by skdcodes** [comment]
@DontAskM8 No, this is an expired version of akamai. It will give us some clue to reverse engineer. 

The current version implemented is v`1.6`. However, this is for v`1.43` (https://github.com/godtoy/akamai-web-sensor-mock/blob/2b66ffc134a8d76cd733cabeb10541ced14164ac/web.akamai.ts#L64). You can get the version of Akamai implemented on a webite using: `bmak.ver;` on the website's console.
